superset/viz.py

```diff
@@ -972,9 +972,10 @@ def _nest(self, metric: str, df: pd.DataFrame) -> List[Dict[str, Any]]:
         if nlevels == 1:
             result = [{"name": n, "value": v} for n, v in zip(df.index, df[metric])]
         else:
+            this_level_index = {i[0] for i in df.index}
```

Comment:
Should this be a `set` or a `list`? Sets don't preserve ordering, without full context, it feels like ordering may matter here (?)

Reply:
The order doesn't seem to matter, because the front end reorders by size.
But I thought it might be better to keep it in order, so I added a few lines